src/main/xtend/delight/nashornsandbox/internal/NashornSandboxImpl.xtend

```diff
@@ -42,33 +48,36 @@ class NashornSandboxImpl implements NashornSandbox {
 		}
 		
 		scriptEngine.eval("\n" +
-                "quit = function() {};\n" +
-                "exit = function() {};\n" +
-                "\n" +
-                "print = function() {};\n" +
-                "echo = function() {};\n" +
-                "\n" +
-                "readFully = function() {};\n" +
-                "readLine = function() {};\n" +
-                "\n" +
-                "load = function() {};\n" +
-                "loadWithNewGlobal = function() {};\n" +
-                "\n" +
-                //"Java = null;\n" +
-             //   "org = null;\n" +
-             //   "java = null;\n" +
-             //   "com = null;\n" +
-             //   "sun = null;\n" +
-             //   "net = null;\n" +
-                "\n" +
-                "$ARG = null;\n" +
-                "$ENV = null;\n" +
-                "$EXEC = null;\n" +
-                "$OPTIONS = null;\n" +
-                "$OUT = null;\n" +
-                "$ERR = null;\n" +
-                "$EXIT = null;\n" +
-                "")
+				(if (!this.allowPrintFunctions) "" +
+						"quit = function() {};\n" +
+						"exit = function() {};\n"
```

Comment:
Why did you set the values to `function() {}` instead of `null`, is there any specific reason?